common/predictive-text/worker-interface.d.ts

```diff
@@ -18,7 +18,7 @@ interface PostMessageOptions {
 /** This Web Workers API interface represents a background task that can be easily created and can send messages back to its creator. Creating a worker is as simple as calling the Worker() constructor and specifying a script to be run in the worker thread. */
 interface Worker {
   onmessage: ((this: Worker, ev: MessageEvent) => any) | null;
-  postMessage(message: any, transfer: Transferable[]): void;
+  postMessage(message: any, transfer: any[]): void;
   postMessage(message: any, options?: PostMessageOptions): void;
```

Comment:
Once again, are these method overloads strictly necessary? What's the minimal interface required to be a valid `WebWorker` and provide the functionality required by the `LMLayer` worker?

Is this sufficient?

```suggestion
  postMessage(message: any): void;
```